wolox-books-angularjs/src/app/components/centered/sign-up/sign-up.controller.js

```diff
@@ -0,0 +1,9 @@
+angular.module('app').controller('SignUpController', ['$scope', '$state',
+  function ($scope, $state) {
+    this.submit = () => {
+      if ($scope.userForm.$valid && $scope.user && $scope.user.password === $scope.user.confirmPassword) {
+        $state.transitionTo('centered.login');
```

Comment:
Where is the api integration?

Reply:
API integration not contemplated in this card. It will be integrated in the next one